extension/src/paymentHandler/make-payment.handler.js

```diff
@@ -0,0 +1,40 @@
+const { makePayment } = require('../web-component-service')
+const pU = require('./payment-utils')
+const c = require('../config/constants')
+
+async function execute (paymentObject) {
+  const makePaymentRequestObj = JSON.parse(paymentObject.custom.fields.makePaymentRequest)
+  // todo: check amount planned
+  const { request, response } = await makePayment(makePaymentRequestObj)
+  const actions = [
+    pU.createAddInterfaceInteractionAction({
+      request, response, type: c.CTP_INTERACTION_TYPE_MAKE_PAYMENT
+    }),
+    pU.createSetCustomFieldAction(c.CTP_CUSTOM_FIELD_MAKE_PAYMENT_RESPONSE, response)
+  ]
+
+  // eslint-disable-next-line default-case
+  switch (response.resultCode) {
```

Comment:
https://docs.adyen.com/checkout/payment-result-codes 
Should we consider the other result codes like,  
AuthenticationFinished, AuthenticationNotRequired which also could indicate a complete payment 🤔

Reply:
I checked this document and only 4 result codes are the final states: https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/v52/payments__resParam_resultCode.  Additionally this document says similar thing: https://docs.adyen.com/development-resources/test-cards/result-code-testing/adyen-response-codes. I think the other result codes are not final and they transform to other states.